plugins/generator-1.16.4/forge-1.16.4/block.definition.yaml

```diff
@@ -4,7 +4,7 @@ templates:
 
   # Block item templates
   - template: json/block_item.json.ftl
-    condition: "renderType() #?= 10,11,12,110,120,2"
+    condition: "renderType() #?= 10,11,12,14,110,120,2"

```

Comment:
Is this line intentionally left out in 1.15.2 generator?

Reply:
Nope, fixed now

Reply:
Ok. I will do another review asap.